app/assets/javascripts/discourse/app/components/composer-editor.hbs

```diff
@@ -1,3 +1,7 @@
+{{!-- TODO(@keegan):
+Remove @editingPost={{this.composer.editingPost}}
+once we add edit/draft support to form templates
+--}}
```

Comment:
Nitpicking, but I'm not sure we need this comment. The one on the JS file is probably enough.